### ISSUE TITLE
mbrola not working

### ISSUE THREAD

**Issue by metal3d** [espeak-ng/espeak-ng#621]
Hi,
First of all: https://github.com/espeak-ng/espeak-ng/blob/master/docs/mbrola.md => you says "Linux installation" but it's not - it's "Debian" installation. For Linux installation, we need something more "global" and universal.

So, now :)

On Fedora. I decided to compile espeak-ng myself (it seems that the provided package in repository is not compiled with mbrola support)

I did:
```
./autogen.sh
./configure --with-mbrola
make
sudo make install
```

Pcaudiolib is installed, everything is ok.

I now compiled MBROLA myself from https://github.com/numediart/MBROLA

```
make
sudo cp Bin/mbrola /usr/local/bin/mbrola
```

OK.

I now tried to download and install voices from https://github.com/numediart/MBROLA-voices
Tried:
- `sudo cp -r data/* /usr/local/share/espeak-ng-data/mbrola` (with directory created)
- in `/usr/share/mbrola`
- with something that worked weeks ago: 
```
sudo mkdir -p /usr/share/mbrola/voices
for voice in data/*; do
    voice=$(basename $voice)
    sudo cp data/$voice/$voice /usr/share/mbrola/voices
done
```

Wherever I put voices, espeak-ng doesn't see voices, and cannot speak with mbrola voice:
```
 espeak-ng -v mb-fr1 "Coucou"
mbrola: Fatal error in line:
mbrola: 
mbrowrap error: mbrola exited with status 253
```

So, maybe it's mbrola, maybe not... 

To be sure, I tried from MBROLA voices dir:
```
espeak-ng -v mb-fr4 "Une phrase pour tester" --pho -s 110 | mbrola fr4/fr4 - out.wav
paplay out.wav 
```
And it works.

So, the problem is to find mbrola voices - any help ?

**Comment by metal3d** [comment]
OK... I make a `git bisect` on mbrola. The last commit breaks things.
I still not see "mb" voices with `espeak-ng --voices` but `espeak-ng -v mb-fr2 "Une phrase pour tester"` is working.

So, the only problem is to know why I cannot list mb voices now.

**Comment by metal3d** [comment]
OK... forget...
It seems that the problems come from the bad mbrola commit. Reverted and reinstalled is ok.

**Comment by valdisvi** [comment]
Can you fix that latest now MBROLA is working for now?
I fixed one found regression in it:
https://github.com/valdisvi/MBROLA/commit/3b77613e8675a09106d13e4b06aefaffdabf79ec

**Comment by GHPS** [comment]
I had the same issue with mbrola
```
mbrola: Fatal error in line:
mbrola: 
mbrowrap error: mbrola exited with status 253
```
The problem has been fixed in commit 3b77613e8675a091.
So this issue can be re-closed.

**Comment by valdisvi** [comment]
Thanks for your effort!
